<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineOptions</class>
 <widget class="QWidget" name="LineOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>X:</string>
     </property>
     <property name="buddy">
      <cstring>x_column</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="legend_label">
     <property name="whatsThis">
      <string>The label the line has in the legend.
Leave blank to prevent this line from appearing in the legend.</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="marker_format">
     <property name="currentText">
      <string>None</string>
     </property>
     <property name="currentIndex">
      <number>9</number>
     </property>
     <item>
      <property name="text">
       <string>Circle          ⬤</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Triangle      ▲</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square        ■</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pentagon    ⬟</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hexagon     ⬢</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Star            ★</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cross          +</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Point           ●</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pixel            ·</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Label</string>
     </property>
     <property name="buddy">
      <cstring>legend_label</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Marker:</string>
     </property>
     <property name="buddy">
      <cstring>marker_format</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QWidget" name="marker_size_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Marker Size</string>
        </property>
        <property name="buddy">
         <cstring>marker_size</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="marker_size">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="y_column"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QWidget" name="line_width_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Line Width</string>
        </property>
        <property name="buddy">
         <cstring>line_width</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="line_width">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="line_format">
     <item>
      <property name="text">
       <string>Solid           —</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dashed      - -</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dotted       ··</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dash-dot   -·</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string extracomment=".">None</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Y:</string>
     </property>
     <property name="buddy">
      <cstring>y_column</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QFrame" name="color_frame">
     <property name="whatsThis">
      <string>Sets the line color.
The current color should appear as
an outline surrounding the button.</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="set_color">
        <property name="text">
         <string>Set Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="x_column"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Line:</string>
     </property>
     <property name="buddy">
      <cstring>line_format</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>legend_label</tabstop>
  <tabstop>x_column</tabstop>
  <tabstop>y_column</tabstop>
  <tabstop>line_format</tabstop>
  <tabstop>marker_format</tabstop>
  <tabstop>set_color</tabstop>
  <tabstop>line_width</tabstop>
  <tabstop>marker_size</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
